sdk/core/Azure.Core/samples/Diagnostics.md

```diff
@@ -86,6 +86,63 @@ using AzureEventSourceListener listener = new AzureEventSourceListener(
 When targeting .NET Standard 2.1, .NET Core 2.2, or newer, you might instead use `e.TimeStamp` to log the time the event was written instead of rendered, like above. It's in UTC format, so if you want to log the local time like in the example call `ToLocaleTime()` first.
 For help diagnosing multi-threading issues, you might also log `e.OSThreadId` which is also available on those same targets.
 
+## ActivitySource support
+
+Azure SDKs released after October 2021 have experimental ActivitySource support. ActivitySource is a simplified way to create and listen to activities added in .NET 5.
+Azure SDKs produce the following kinds of Activities:
+
+- *HTTP calls*: every HTTP call originating from Azure SDKs
+- *client method calls*: for example, `BlobClient.DownloadTo` or `SecretClient.StartDeleteSecret`.
+- *messaging events*: EventHubs and ServiceBus message creation is traced and correlated with its sending, receiving, and processing.
```

Comment:
```suggestion
- *messaging events*: Event Hubs and Service Bus message creation is traced and correlated with its sending, receiving, and processing.
```

To be pedantic, Event Hubs messages are not correlated for receiving.  It was defined as optional and "low-value" in the spec, and @lmolkova has offered some strong rationale why adding it isn't a good idea.

I'm not sure that it is important to make that distinction here, but I wanted to mention it, just in case.

Reply:
Hm, we still create spans for Receive client calls, don't we?

Reply:
In Service Bus yes,  in Event Hubs no.

Reply:
Huh, TIL :)